src/main/python/trec-covid/remove_judged_docids.py

```diff
@@ -54,6 +52,9 @@ def main():
                 score = float(cols[4])
                 tag = cols[5]
 
+                if args.runtag is not None:
```

Comment:
The code below is preferred as it covers both `runtag == None` and `runtag == ''`
```suggestion
                if args.runtag:
```

Reply:
Oh wow, good to know.